Integrations/MailSenderNew/MailSenderNew.yml

```diff
@@ -25,11 +25,14 @@ configuration:
   name: fqdn
   required: false
   type: 0
-- defaultvalue: 'false'
-  display: Use TLS for connection
+- display: Use TLS for connection
```

Comment:
Please use display name `Connection Security` with options:
* None
* STARTTLS
* SSL/TLS

By default None should be selected.